### DISCUSSION TITLE
Abnormal behaviour in connections

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx#10016] [General]
Having some abnormal behaviour in connections in emqx v5.0.11, cluster is having around 3000 connections, tried to kick out clients but all are reconnection immediately. Tried sending diconnect packets, response - Couldnt disconnect connection. Also tried restarting the cluster still reconnection is happening. Another strange thing is that many of the clients are connected with the same client id
What is the reason for this?
Anyway to disconnect all connections?

Thanks in advance......

**Comment by lafirest** [comment]
@alexa22013 
Hi, Sorry, but I think there are some bugs in your client code.
The `reconnect` and `connect with the same clientid` both are client behaviour.
